spanner/row_test.go

```diff
@@ -399,7 +400,8 @@ func TestNullTypeErr(t *testing.T) {
 			&dt,
 		},
 	} {
-		wantErr := errDecodeColumn(ntoi(test.colName), errDstNotForNull(test.dst))
+		var wantErr error
```

Comment:
This looks a bit weird but is probably fine since the less we have to diverge from upstream the better.